src-ui/src/app/components/admin/settings/settings.component.html

```diff
@@ -1,4 +1,9 @@
-<pngx-page-header title="Settings" i18n-title>
+<pngx-page-header
+  title="Settings"
+  i18n-title
+  info="Options to customize appearance, notifications, saved views and more. Settings apply to the current user only."
```

Comment:
Do you want to emphasize this is the current user?  As how the config indicates it applies to all?

Reply:
Ha, yea funny enough I had that and removed it. Sure!